scripts/dappnode_install_pre.sh

```diff
@@ -20,25 +19,33 @@ detect_installation_type() {
     fi
 }
 
-
 add_docker_repo() {
```

Comment:
same as above, we should be using something like https://github.com/docker/docker-install/tree/master